assets/javascripts/modules/SSO_links.js

```diff
@@ -25,7 +25,7 @@ module.exports = function(element, ssoUrl, ssoMethod) {
   var winId;
   var openInNewWindow;
   var allowLinkCLickEvent = true;
-  var elementHref;
+  var elementActionOrHref;
```

Comment:
this could just become `destinationUrl` to match the property in the destination object?